src/devtools/useAtomDevtools.ts

```diff
@@ -31,11 +30,28 @@ type Extension = {
   connect: (options?: Config) => ConnectionResult
 }
 
+const isWritable = <Value, Update, Result extends void | Promise<void>>(
+  atom: Atom<Value> | WritableAtom<Value, Update, Result>
+): atom is WritableAtom<Value, Update, Result> =>
+  !!(atom as WritableAtom<Value, Update, Result>).write
+
 export function useAtomDevtools<Value>(
   anAtom: WritableAtom<Value, Value>,
   name?: string,
   scope?: Scope
-) {
+): void
+
+export function useAtomDevtools<Value>(
+  anAtom: Atom<Value>,
+  name?: string,
+  scope?: Scope
+): void
```

Comment:
Now, I'm not sure if we need function overload for `useAtomDevtools`.
If not, let's remove it.

Reply:
Did you see any concern removing the function overload?
Only what is useful would be explicit typing.